suggestions/cloud_services.json

```diff
@@ -1 +1,30 @@
-[]
\ No newline at end of file
+[
+    {
+        "id": "com.owncloud.android",
+        "label": "ownCloud",
+        "source": "fg",
+        "reason": "Add an ownCloud server now and get... Your Cloud, Your Data, Your Way!\nhttps://owncloud.org/",
+        "repo": "https://github.com/owncloud/android"
+    },
```

Comment:
This cannot be included because ownCloud enterprise edition is a non-free software.

Reply:
Understood! Will be removed in the next commits ✅